### ISSUE TITLE
HP:6001359	Subcutaneous abscess (old label)	Cholesterol gallblader polyp (new label)

### ISSUE THREAD

**Issue by grahamponting** [obophenotype/human-phenotype-ontology#11370]
Hi,
HP:6001359 Subcutaneous abscess (old label), Cholesterol gallblader polyp (new label)
I am concerned about the above change in the list distributed on 24th November, indicating a name change as above. It seems the same ID is being reused for a completely different concept ?
This could lead to confusion in end users using an different  versions of HPO.

Graham Ponting

**Comment by pnrobinson** [comment]
@grahamponting 
Thanks very much for pointing out this possible issue! I appreciate the attention to detail!!!
In this case, I am not sure where the error was.
These was this issue:
https://github.com/obophenotype/human-phenotype-ontology/issues/10687
saying

I noticed we already have this:
[Subcutaneous abscess HP:6001359](https://hpo.jax.org/browse/term/HP:6001359)
But the link to Abscess was missing, I have added it.

That link goes out to Cholesterol gallblader polyp (which was recently created as a new term).

[Subcutaneous abscess HP:0020116](https://hpo.jax.org/browse/term/HP:0020116) was also created at about this time as a new term.

At present, I think the error was in https://github.com/obophenotype/human-phenotype-ontology/issues/10687, not in the HPO files. Could I ask if you have observed this item in a released HPO file? I could not just now.

**Comment by pnrobinson** [comment]
As far as I can see, there must have been some copy paste error for the comment made on the tracker, there is no error to be found in the actual HPO files. I will close, but thanks very much for checking!
